robot/Cumulus/tests/browser/donations_payments/create_donation_and_payment.robot

```diff
@@ -2,108 +2,61 @@
-    ${pay_id}    Return Locator Value    payments.field-value    Payment Number
-    Click Modal Button    Save
-    Wait Until Modal Is Closed
-    Click Special Related List Button  Payments    Schedule Payments
-    Wait For Locator    frame    Create one or more Payments for this Opportunity
-    Choose Frame    Create one or more Payments for this Opportunity
-    Verify Field Values
-    ...    Payment Writeoff Amount=$250.00
-    ...    Remaining Balance=$750.00
-    Click Button    Cancel
-    Current Page Should Be     Details    Opportunity
-    Select Tab    Related
-    Click First Matching Related Item Popup Link    Payments    Unpaid    Edit
-    Wait Until Modal Is Open
-    Set Checkbutton To    Written Off    checked
-    Set Checkbutton To    Paid           checked
-    Click Modal Button    Save
-    Page Should Contain    A Payment can't be both paid and written off. You must deselect one or both checkboxes.
-    Click Modal Button    Cancel
-    Current Page Should Be     Details    Opportunity
-    Click ViewAll Related List    Payments
-    Verify Details
-    ...    Unpaid=3
-    ...    Written Off=1
-    
-Verify values in Writeoff Remaining Balance Page
-    [tags]  unstable
-    Go To Page     Details    Opportunity    object_id=&{opportunity}[Id]
-    Select Tab    Related
-    Click First Matching Related Item Popup Link    Payments    Unpaid    Edit
-    Wait Until Modal Is Open  
-    Set Checkbutton To    Written Off    checked
-    Populate Field    Payment Amount    200 
-    Click Modal Button    Save
-    Wait Until Modal Is Closed
-    Click Related List Dd Button    Payments    Show one more action    Write Off Payments 
-    Wait For Locator    frame    Write Off Remaining Balance
-    Choose Frame    Write Off Remaining Balance 
-    Wait Until Page Contains    You are preparing to write off 2 Payment(s)
-    Verify Field Values
-    ...    Payment Writeoff Amount=$450.00
-    ...    Remaining Balance=$550.00  
-    Page Should Contain    You are preparing to write off 2 Payment(s) totaling $550.00
-    Choose Frame    Write Off Remaining Balance
-    Click Button    Cancel
-    Current Page Should Be     Details    Opportunity
-    Select Tab    Related
-    Load Related List    Payments
-    Click ViewAll Related List    Payments
-    Verify Details
-    ...    Unpaid=2
-    ...    Written Off=2
\ No newline at end of file
+
+    Go To Page                           Details
+    ...                                  Opportunity
+    ...                                  object_id=${data}[contact_opportunity][Id]
+
+    Select Tab                           Related
+    Load Related List                    Payments
+    Click Special Related List Button    Payments               Schedule Payments
+    Current Page Should Be               Custom                 SchedulePayment
+
+    ${date} =                            Get Current Date       result_format=%-m/%-d/%Y
```

Comment:
Can this be moved to as a part of `initialize test data`?

Reply:
done